src/runtime/tests/manifest-parser-test.ts

```diff
@@ -71,6 +71,20 @@ describe('manifest parser', () => {
         h1: create 'my-id' @tiedToRuntime
         h2: create #mytag @tiedToArc`);
   });
+  it('parses schema annotations', () => {
+    parse(`
+      schema Abcd
+        hardRef: &Package @hardRef
```

Comment:
what is`@Package` annotation? i can't find where it is defined.

Reply:
changing the annotation to @aFoo to make it clear it is not a special annotation defined anywhere